server/channels/app/post_test.go

```diff
@@ -731,53 +729,6 @@ func TestImageProxy(t *testing.T) {
 	}
 }
 
-func TestMaxPostSize(t *testing.T) {
-	t.Parallel()
-
-	testCases := []struct {
```

Comment:
Hmm, that is actually being tested in the store layer, do you think we should have that here as well? 

See: https://github.com/mattermost/mattermost/blob/e3a0a403938f7f503cf729960fb01d7f28866bec/server/channels/store/storetest/post_store.go#L4258

Reply:
That's a weird one -- why is it testing it twice..
Anyway, just want to make sure, does that test cover the cases here?

Reply:
Judging by the code in the store layer, I think there is no case that `determineMaxPostSize()` can return less than `PostMessageMaxRunesV2` so I'm not sure if it would make any difference to test it on the app level. Since we couldn't figure out the double test, I'm removing the second run.

Reply:
@isacikgoz I am still struggling to understand what makes the test pass given we check for equality, but I'd expect the DB to have higher defaults than `model.PostMessageMaxRunesV2`.

Reply:
Oh, well, scratch that. V2 is `65535` bytes, so I guess ti makes sense.